ormar/queryset/queryset.py

```diff
@@ -894,7 +894,6 @@ async def first(self, *args: Any, **kwargs: Any) -> "T":
         Gets the first row from the db ordered by primary key column ascending.
 
         :raises NoMatch: if no rows are returned
-        :raises MultipleMatches: if more than 1 row is returned.
```

Comment:
Why did you remove it - it does indeed check for one row only in `self.check_single_result_rows_count(processed_rows)`, please restore it.